Tests/FoundationTests/URLSessionExtensionsTests.swift

```diff
@@ -0,0 +1,41 @@
+// URLSessionExtensionsTests.swift - Copyright 2023 SwifterSwift
+
+@testable import SwifterSwift
+import XCTest
+
+#if canImport(Foundation)
+import Foundation
+
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
+
+final class URLSessionExtensionsTests: XCTestCase {
+    func testDataSync() {
+        let gemfileContent = "source 'https://rubygems.org'\n\n" +
+            "gem 'xcpretty'\n" +
+            "gem 'danger'\n" +
+            "gem 'danger-swiftlint'\n" +
+            "gem 'danger-xcode_summary'\n" +
+            "gem 'xcpretty-json-formatter'\n\n" +
+            "gem 'fastlane'\n" +
+            "gem 'cocoapods'"
+        let url = URL(string: "https://raw.githubusercontent.com/SwifterSwift/SwifterSwift/5.3.0/Gemfile")!
+        var data: Data!
+        var response: URLResponse!
```

Comment:
You need to assert that they're non-nil.
```suggestion
        var data: Data?
        var response: URLResponse?
```